src/webhooks/index.ts

```diff
@@ -1,3 +1,5 @@
+import '@sentry/tracing';
```

Comment:
Any idea why this is needed? Is it needed? I know we had this previously, but I am not sure it is needed.

Reply:
In order to initialize Sentry this has to be called